mapfishapp/src/main/webapp/app/addons/bangeocoder/README.md

```diff
@@ -0,0 +1,110 @@
+BAN Geocoder ADDON
+==================
+
+**** This addon allows user to search adress from BAN service in France. 
+Ex : https://adresse.data.gouv.fr/api/
+
+**** This addon use autocompletion service (GET method) with 3 parameters : 
+ service URL : http://api-adresse.data.gouv.fr/search/?
+ search value : "q=8 bd du port"
+ limit of responses (is set to 5 by default) : "limit=5"  
+Ex : http://api-adresse.data.gouv.fr/search/?q=8 bd du port&limit=5
+
+**** The service return GeoJSON like that :
+
+    /!\ Default SRS code is EPSG:4326 /!\
+
+{
```

Comment:
please format the js/json according to the standards (eg http://jsbeautifier.org/)